app/subscriber/src/features/content/view-content/ViewContent.tsx

```diff
@@ -207,15 +207,15 @@ export const ViewContent: React.FC = () => {
               content?.contentType === ContentTypeName.Story
             }
           >
-            <span>{parse(content?.body ?? '')}</span>
+            <span>{parse(content?.body?.replace(/\n+/g, '<br><br>') ?? '')}</span>
```

Comment:
Do not do this here.  Fix the issue.  All this will do is hide it until we display the same text somewhere else.

Reply:
The similar mechanism is currently being applied in the Editor side for this. The backend service already did its job for the format of the text, and as mentioned the same text could be displayed in different places, which the css styles could be different based on what the text will be displayed on, such as just showing the text in the span/div tag here, or the text could be shown in the WYSIWYG editor, which the style would be different, so there will be no fix-for-all in this case. Thanks

Reply:
I suppose having a newline in the DB is appropriate in many cases.  However, most of our content would be native HTML.  This implementation will result future display issues.  It's not a great solution.  For example our reports use HTML.  The output will continue to have issues.

Secondly a double break `<br><br>` isn't great either.  Although I do understand it's a lot simpler than trying to surround a section in a paragraph tag `<p>...</p>`.